pep-0692.rst

```diff
@@ -463,13 +534,15 @@ previous example::
     >>> def foo(**kwargs: **Movie): ...
     ...
     >>> foo.__annotations__
-    {'kwargs': **Movie}
+    {'kwargs': Unpack[Movie]}
 
-The double asterisk syntax should call the ``__unpack__`` special method on
-the object it was used on. This means that ``def foo(**kwargs: **T): ...`` is
-equivalent to ``def foo(**kwargs: T.__unpack__()): ...``. In addition,
-``**Movie`` in the example above is the ``repr`` of the object that
-``__unpack__()`` returns.
+To accomplish this, we propose a new dunder called ``__typing_kwargs_unpack__``.
+The double asterisk syntax should result in a call to the
+``__typing_kwargs_unpack__`` special method on an object it was used on.
+This means that ``def foo(**kwargs: **T): ...`` is equivalent to
```

Comment:
```suggestion
This means that at runtime, ``def foo(**kwargs: **T): ...`` is equivalent to
```
Because it isn't the same to static type checkers, presumably.